### DISCUSSION TITLE
Wifi Connection

### DISCUSSION THREAD

**Discussion by eaudiffred** [Ribbit-Network/ribbit-network-frog-hardware#51] [Q&A]
Hello,
I'm having trouble finding the "WiFi Connect" hot spot on any device after powering up the unit.  I've tried restarting the pi a few times and have checked multiple devices.  It looks like the TP link adapter is not transmitting.  When starting up the adapter flashes green quickly for about 3 seconds then stops completely.  According to the user manual, no flashing green light means its not transmitting.  Any trouble shooting ideas?
![image](https://user-images.githubusercontent.com/88562047/134093261-f879f9f8-3e20-4fbd-b4b5-bc8b304a3712.png)

Thanks!

**Comment by keenanjohnson** [comment]
Hmmmmm that is strange @eaudiffred! Do you have an ethernet cord you could plug into the device so I can help you debug it remotely?

![IMG_9638](https://user-images.githubusercontent.com/2559382/134094917-f9fea1c1-3888-4961-9eda-f976f4287002.JPG)
?

**Comment by eaudiffred** [comment]
@keenanjohnson,
I do.  It's plugged in now.

Also, for reference:
- I tried both usb ports on the pi and got the same result.  
- When I plugged it into my laptop, it would slow blink
- I plugged it back into the pi and restarted.  I stood there and watched it for about 5min.  It would randomly blink 3 or 4 times in a row every 3 min or so.

**Comment by keenanjohnson** [comment]
Thanks I can access some debug information here, so let me take a look and
I'll report back! Thanks so much and sorry for the issue!

- Keenan
